src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -48,6 +51,19 @@ public function isGranted(mixed $role, mixed $object = null, ?string $field = nu
         }
     }
 
+    public function isGrantedForUser(UserInterface $user, mixed $attribute, mixed $subject = null, ?string $field = null): bool
+    {
+        if (!$this->userSecurityChecker) {
```

Comment:
What about not registering the function at all in this case? Then you would be able to spot issues at the time the Twig templates are compiled instead of running into exceptions at runtime.

Reply:
I like this idea. I've done this as well.

Reply:
Good idea, especially with the `guard` feature of Twig.